### ISSUE TITLE
Staging Review finding: Segmented button doesn't meet contrast minimums

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#136813]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 1010 Health Apps (33001)
**Product:** Claim Status Tool
**Feature:** Finding Claims

## Findings details

**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The light grey of the segmented button against white (background and active button) has a contrast ratio of 1.3:1, below the required 3:1 to meet WCAG 1.4.11. The light grey against the dark grey is also 1.3:1.

When UI elements/controls don't meet contrast minimums, some users may struggle to understand which control is active, inactive, has focus, etc. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1079" height="679" alt="Image" src="https://github.com/user-attachments/assets/3b878ea2-7f24-4338-a5bd-14710d470eea" />

<img width="1090" height="684" alt="Image" src="https://github.com/user-attachments/assets/eca48641-c5e8-4977-8b21-232774ea6a7e" />

</details>

## Recommended action

Ensure that the states of the segmented button meet the minimum contrast ratio of 3:1. This can be done via using different colors/values, or through the user of borders, etc. 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.4.11 Non-Text Contrast - Level AA](https://www.w3.org/TR/WCAG22/#non-text-contrast)
- **Modality:** N/A
- **Design System Component:** [Button segmented](https://design.va.gov/components/button/button-segmented)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
